Api/Controllers/LevelMetadataController.cs

```diff
@@ -0,0 +1,67 @@
+using Microsoft.AspNetCore.Mvc;
+using ShmupCreator.Contracts;
+using ShmupCreator.Services;
+
+namespace ShmupCreator.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class LevelMetadataController : ControllerBase
+{
+    private readonly ILevelMetadataService _levelMetadataService;
+
+    public LevelMetadataController(ILevelMetadataService levelMetadataService)
+    {
+        _levelMetadataService = levelMetadataService;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateNew([FromBody] CreateLevelMetadataRequest createLevelMetadataRequest)
+    {
+        var levelMetadata = await _levelMetadataService.Create(createLevelMetadataRequest);
+        return Ok(new LevelMetadataResponse
```

Comment:
Mapper can also be used. E.g. return Ok(_mapper.MapToLevelMetadataResponse(levelMetadata)); Same for other endpoints.